context.go

```diff
@@ -217,9 +217,14 @@ func (c *Ctx) detectGOPATH(path string) (string, error) {
 	return "", errors.Errorf("%s is not within a known GOPATH", path)
 }
 
-// ImportForAbs returns the import path for an absolute project path by trimming the
-// `$GOPATH/src/` prefix.  Returns an error for paths equal to, or without this prefix.
-func (c *Ctx) ImportForAbs(path string) (string, error) {
+// ImportPathForProject returns the import path for an absolute project path by trimming the
+// `GOPATH/src/` prefix.  Returns an error for paths equal to, or without this prefix.
+func (c *Ctx) ImportPathForProject(p *Project) (string, error) {
+	path := p.AbsRoot
+	if p.AbsRoot != p.ResolvedAbsRoot {
+		path = p.ResolvedAbsRoot
+	}
```

Comment:
Can't we just do `path = p.ResolvedAbsRoot` instead?

If they are the same, we did what line 223 is doing. If they are different, we did the if block logic.

Reply:
Or just use `p.ResolvedAbsRoot` instead of `path`...